shims/common/src/main/scala/org/apache/gluten/config/GlutenConfig.scala

```diff
@@ -37,97 +38,90 @@ case class GlutenNumaBindingInfo(
-  def enableNativeValidation: Boolean = conf.getConf(NATIVE_VALIDATION_ENABLED)
+  def enableNativeValidation: Boolean = getConf(NATIVE_VALIDATION_ENABLED)
 
-  def enableColumnarBatchScan: Boolean = conf.getConf(COLUMNAR_BATCHSCAN_ENABLED)
+  def enableColumnarBatchScan: Boolean = getConf(COLUMNAR_BATCHSCAN_ENABLED)
 
-  def enableColumnarFileScan: Boolean = conf.getConf(COLUMNAR_FILESCAN_ENABLED)
+  def enableColumnarFileScan: Boolean = getConf(COLUMNAR_FILESCAN_ENABLED)
 
-  def enableColumnarHiveTableScan: Boolean = conf.getConf(COLUMNAR_HIVETABLESCAN_ENABLED)
+  def enableColumnarHiveTableScan: Boolean = getConf(COLUMNAR_HIVETABLESCAN_ENABLED)
 
   def enableColumnarHiveTableScanNestedColumnPruning: Boolean =
-    conf.getConf(COLUMNAR_HIVETABLESCAN_NESTED_COLUMN_PRUNING_ENABLED)
+    getConf(COLUMNAR_HIVETABLESCAN_NESTED_COLUMN_PRUNING_ENABLED)
 
-  def enableVanillaVectorizedReaders: Boolean = conf.getConf(VANILLA_VECTORIZED_READERS_ENABLED)
+  def enableVanillaVectorizedReaders: Boolean = getConf(VANILLA_VECTORIZED_READERS_ENABLED)
 
-  def enableColumnarHashAgg: Boolean = conf.getConf(COLUMNAR_HASHAGG_ENABLED)
+  def enableColumnarHashAgg: Boolean = getConf(COLUMNAR_HASHAGG_ENABLED)
 
-  def forceToUseHashAgg: Boolean = conf.getConf(COLUMNAR_FORCE_HASHAGG_ENABLED)
+  def forceToUseHashAgg: Boolean = getConf(COLUMNAR_FORCE_HASHAGG_ENABLED)
 
-  def mergeTwoPhasesAggEnabled: Boolean = conf.getConf(MERGE_TWO_PHASES_ENABLED)
+  def mergeTwoPhasesAggEnabled: Boolean = getConf(MERGE_TWO_PHASES_ENABLED)
 
-  def enableColumnarProject: Boolean = conf.getConf(COLUMNAR_PROJECT_ENABLED)
+  def enableColumnarProject: Boolean = getConf(COLUMNAR_PROJECT_ENABLED)
 
-  def enableColumnarFilter: Boolean = conf.getConf(COLUMNAR_FILTER_ENABLED)
+  def enableColumnarFilter: Boolean = getConf(COLUMNAR_FILTER_ENABLED)
 
-  def enableColumnarSort: Boolean = conf.getConf(COLUMNAR_SORT_ENABLED)
+  def enableColumnarSort: Boolean = getConf(COLUMNAR_SORT_ENABLED)
 
-  def enableColumnarWindow: Boolean = conf.getConf(COLUMNAR_WINDOW_ENABLED)
+  def enableColumnarWindow: Boolean = getConf(COLUMNAR_WINDOW_ENABLED)
 
-  def enableColumnarWindowGroupLimit: Boolean = conf.getConf(COLUMNAR_WINDOW_GROUP_LIMIT_ENABLED)
+  def enableColumnarWindowGroupLimit: Boolean = getConf(COLUMNAR_WINDOW_GROUP_LIMIT_ENABLED)
 
-  def veloxColumnarWindowType: String = conf.getConfString(COLUMNAR_VELOX_WINDOW_TYPE.key)
+  def veloxColumnarWindowType: String = getConf(COLUMNAR_VELOX_WINDOW_TYPE)
 
-  def enableColumnarShuffledHashJoin: Boolean = conf.getConf(COLUMNAR_SHUFFLED_HASH_JOIN_ENABLED)
+  def enableColumnarShuffledHashJoin: Boolean = getConf(COLUMNAR_SHUFFLED_HASH_JOIN_ENABLED)
 
   def shuffledHashJoinOptimizeBuildSide: Boolean =
-    conf.getConf(COLUMNAR_SHUFFLED_HASH_JOIN_OPTIMIZE_BUILD_SIDE)
+    getConf(COLUMNAR_SHUFFLED_HASH_JOIN_OPTIMIZE_BUILD_SIDE)
 
-  def enableNativeColumnarToRow: Boolean = conf.getConf(COLUMNAR_COLUMNAR_TO_ROW_ENABLED)
```

Comment:
ok, we doesn't use this config, we should remove all of them in sperated pr.

Reply:
yes.